buildSrc/src/main/kotlin/datadog/gradle/plugin/version/GitDescribeValueSource.kt

```diff
@@ -0,0 +1,67 @@
+package datadog.gradle.plugin.version
+
+import org.gradle.api.GradleException
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.ValueSource
+import org.gradle.api.provider.ValueSourceParameters
+import org.gradle.process.ExecOperations
+import org.gradle.process.ExecResult
+import java.io.ByteArrayOutputStream
+import java.nio.charset.Charset
+import javax.inject.Inject
+
+abstract class GitDescribeValueSource @Inject constructor(
+  private val execOperations: ExecOperations
+) : ValueSource<String, GitDescribeValueSource.Parameters> {
+  override fun obtain(): String {
+    val workDir = parameters.workingDirectory.get()
+    val tagPrefix = parameters.tagVersionPrefix.get()
+
+    val commandsArray = mutableListOf(
+      "git",
+      "describe",
+      "--abbrev=8",
+      "--tags",
+      "--first-parent",
+      "--match=$tagPrefix[0-9].[0-9]*.[0-9]",
```

Comment:
```suggestion
      "--match=$tagPrefix[0-9].[0-9]*.[0-9]*",
```
Don't under estimate how many patches a release can have 😅